ci/tekton/zcash-android-sdk-lint-task.yml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: zcash-android-sdk-lint
+spec:
+  workspaces:
+    - name: venv
+      emptyDir: {}
+  resources:
+    inputs:
+      - name: source
+        resourceSpec:
+          type: git
+          params:
+            - name: revision
+              value: master
+            - name: url
+              value: https://github.com/zcash/zcash-android-wallet-sdk
```

Comment:
A `Task` just needs to defines the required inputs.

```
    inputs:
      - name: source
        type: git
```

The caller will use refSpec to define the input resource parameters.

Same with the other task.

Check with
```
$ kubectl apply -f ci/tekton/zcash-android-sdk-lint-task.yml 
Error from server (BadRequest): error when creating "ci/tekton/zcash-android-sdk-lint-task.yml": admiss
ion webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming n
ew object: json: unknown field "resourceSpec"
```

Reply:
Is the value something I need in the `zcash-android-pr-triggerbinding.yml`?

I'm confused how it knows where to pull from.

Reply:
https://github.com/zcash/zcash-android-wallet-sdk/pull/137/files#diff-ce8c8f8bcaa48783caf6cee9e5fb6d36R15

That's one of the binding parameters. Github sends you the clone url in the PR post. Everything you need is in the post.